mas/Makefile

```diff
@@ -0,0 +1,11 @@
+
+lint:
+	poetry run adev -v -n 0 lint -p .
+
+fmt: 
+	poetry run adev -n 0 fmt -p .
+
+test:
```

Comment:
Note, this is why we need different python and authonomy repos.

by default this will try to lint and format all thirdparty deps as well which is none optimal